pkg/controller/admissionchecks/multikueue/workload_test.go

```diff
@@ -50,6 +51,9 @@ var (
 )
 
 func TestWlReconcile(t *testing.T) {
+	now := time.Now()
+	fakeClock := testingclock.NewFakeClock(now)
```

Comment:
If you provide `now` to NewFakeClock, it means that every time you use fakeClock.Now(), you’ll get that same initial time. However in this line https://github.com/kubernetes-sigs/kueue/blob/f1efd51f4297cf837bb41f739e6dac546ebcc2ea/pkg/controller/admissionchecks/multikueue/workload_test.go#L811 , we're using time.Now() directly, which means the time could be different each time it’s called. The idea, then, is not only to create a `FakeClock` and propagate it but also to ensure the same time is used in tests to prevent flaky results.

Reply:
Else, could we replace time.Now() everywhere with just now ??

Reply:
Yeah, that what I mean.